src/support/generation.c

```diff
@@ -107,9 +107,8 @@ __wt_gen_drain(WT_SESSION_IMPL *session, int which, uint64_t generation)
     struct timespec start, stop;
     WT_CONNECTION_IMPL *conn;
     WT_SESSION_IMPL *s;
-    uint64_t time_diff_ms, v;
+    uint64_t time_diff_ms, v, minutes;
```

Comment:
We generally alphabeticalize anything we can - which makes `minutes` the first thing in the list of local variables here.

Reply:
Oh yes, sure thing. Thanks for pointing that out Alex!